src/main/java/com/rocketden/main/controller/v1/GameController.java

```diff
@@ -0,0 +1,40 @@
+package com.rocketden.main.controller.v1;
+
+import com.rocketden.main.dto.room.GetRoomRequest;
+import com.rocketden.main.dto.room.RoomDto;
+import com.rocketden.main.service.RoomService;
+import com.rocketden.main.dto.game.StartGameRequest;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class GameController extends BaseRestController {
+    
+    private final RoomService service;
+
+    @Autowired
+    public GameController(RoomService service) {
+        this.service = service;
+    }
+
+    @PostMapping("/start")
+    public ResponseEntity<Void> startGame(@RequestBody StartGameRequest request) {
+        // Check if current user ID is the host
+        GetRoomRequest getRoomRequest = new GetRoomRequest();
+        getRoomRequest.setRoomId(request.getRoomId());
+        RoomDto roomDto = service.getRoom(getRoomRequest);
+
+        if (!roomDto.getHost().getNickname().equals(request.getUser().getNickname())) {
```

Comment:
This is throwing an error for me, same as it did for you yesterday. (See images below.)

Can you upload a screencast of it working? That way I can see what it's supposed to look like, and also troubleshoot why it may be doing this (since I remember this happened for you occasionally as well).

<img width="566" alt="Backend 405 Error" src="https://user-images.githubusercontent.com/7987279/95028947-2b1b5680-0659-11eb-9f52-6fb92c26c44c.png">

<img width="364" alt="Frontend Error" src="https://user-images.githubusercontent.com/7987279/95028956-5140f680-0659-11eb-8ea9-7b76d73c3bc2.png">

Reply:
Screencast uploaded! It's in the description of the PR.

Reply:
405 error fixed -- turns out the problem was typo in the handleClick function :)